helm_deploy/keyworker-api/templates/housekeeping-cronjob.yaml

```diff
@@ -15,10 +15,10 @@ spec:
       template:
         spec:
           containers:
-            - name: keyworker-api
+            - name: housekeeping-keyworker-api
               image: ghcr.io/ministryofjustice/hmpps-devops-tools
               args:
                 - /bin/sh
                 - -c
-                - curl --retry 2 -XPUT http://keyworker-api/queue-admin/retry-all-dlqs
+                - curl --retry 2 -XPUT http://keyworker-api/batch/queue-admin/retry-all-dlqs
```

Comment:
I don't think you need to update this URL - it isn't managed by `BatchController`

Reply:
Done